lib/jsdom/browser/Window.js

```diff
@@ -339,7 +336,11 @@ module.exports = function Window(options) {
       });
     }
   });
-};
+}
+
+inherits(dom.EventTarget, Window, dom.EventTarget.prototype);
```

Comment:
Yes, for some reason our inherits doesn't include the methods otherwise, we should maybe have a look at that too.